src/scythe-bidder/constants.ts

```diff
@@ -18,11 +18,53 @@ export const factions = [
   "Rusviet",
 ] as const;
 
+export const mats_base = [
```

Comment:
Let's keep the convention of using camelCase or ALL_CAPS for constants when possible. This actually might make sense as ALL_CAPS.

Reply:
I agree with using ALL_CAPS. Also, we can get rid of these constants if we pass the allowed combos through setupData. Let me see how it works.

Reply:
Also, I want to remove the the `mats` and `factions` variables and replace them with `MATS_IFA` and `FACTIONS_IFA`. Any suggestions for the variable names?